app/models/manageiq/providers/azure/event_catcher_mixin.rb

```diff
@@ -0,0 +1,38 @@
+module ManageIQ::Providers::Azure::EventCatcherMixin
+  def parse_event_type(event)
+    event_name = event["eventName"]["value"]
+    event_type = ""
+
+    unless event["authorization"].nil? || event["authorization"]["action"].nil?
```

Comment:
The latter is better, because parse_event_action has to fetch that path again...might as well only do it once in the parse_event_action method.